cmd/template/cluster/testdata/run_template_cluster_capa_2.golden

```diff
@@ -0,0 +1,123 @@
+---
+apiVersion: v1
+data:
+  values: |
+    aws:
+      awsClusterRole: default
+      region: the-region
+    bastion: {}
+    clusterDescription: just a test cluster
+    clusterName: test1
+    controlPlane:
+      instanceType: control-plane-instance-type
+      replicas: 3
+    machinePools:
+    - availabilityZones:
+      - eu-west-1a
+      - eu-west-1b
+      customNodeLabels:
+      - label=value
+      instanceType: big-one
+      maxSize: 5
+      minSize: 2
+      name: worker1
+      rootVolumeSizeGB: 200
+    network:
+      apiMode: private
+      dnsMode: private
+      subnets:
+      - cidrBlock: 10.123.0.0/18
+      - cidrBlock: 10.123.64.0/18
+      topologyMode: GiantSwarmManaged
+      vpcCIDR: 10.123.0.0/16
+      vpcMode: private
+    organization: test
+    proxy:
+      enabled: true
+      http_proxy: http://internal-a1c90e5331e124481a14fb7ad80ae8eb-1778512673.eu-west-2.elb.amazonaws.com:4000
+      https_proxy: https://internal-a1c90e5331e124481a14fb7ad80ae8eb-1778512673.eu-west-2.elb.amazonaws.com:4000
+      no_proxy: test-domain.com
```

Comment:
I know this is not the right place, but since I'm noticing it here: we normally use camelCase for the values properties. The use of snake_case here is an outlier.

Reply:
I see, but we have already been using cluster templates in this format.